libs/galley-types/src/Galley/Types.hs

```diff
@@ -493,6 +527,7 @@ instance FromJSON Conversation where
                     <*> o .:  "type"
                     <*> o .:  "creator"
                     <*> o .:  "access"
+                    <*> o .:  "access_role"
```

Comment:
It's safe to assume that `access_role` will always be present after `galley` is deployed with this change. However, perhaps to remove coupling of other services to `galley-types` it would be safer to change this to
`o .:? "access_role" .!= VerifiedAccessRole` (i.e., same as we do on the `Data` module)

Reply:
Ah, that makes sense, yes.